### PR TITLE
Experimental Feature: Game Screenshots

### PR DESCRIPTION
**Pull request by LitRitt**

### CODE REVIEW

File: Delta/Emulation/GameViewController.swift
Comment (rileytestut) [inline Delta/Emulation/GameViewController.swift:1095]: See my note below, this is a potential bug.

**Review by rileytestut — CHANGES_REQUESTED** [review]
Thanks for breaking these PRs up! Overall looks good to me, just have one note about the implementation of `isAuthorized` that could result in minor bug, but once that’s fixed I’d be happy to merge 🙂

Also I appreciate you putting the PHPhotoLibrary stuff in an extension! The more isolated changes are from the rest of the app, the better 👍

**Review by rileytestut — APPROVED** [review]
Nice approach, passing in a closure that's only run if authorized is a good idea. 

Thanks for these changes, everything looks good to me! Will merge this in along with #241 once everything is merged into `develop` branch 👍

**Review by takashiou — COMMENTED** [review]
好👍

**Review by jesn459 — COMMENTED** [review]
Gracias

**Pull request merged**
